src/API/Configurations.js

```diff
@@ -0,0 +1,90 @@
+import { useQueryClient } from 'react-query';
+
+import { useOkapiQuery } from './useOkapiQuery';
+import { useOkapiMutation } from './useOkapiMutation';
+
```

Comment:
is it new style to have 2 empty lines?

Reply:
General stripes lint rules added this possibility a while ago.
See https://folio-project.slack.com/archives/C210UCHQ9/p1607449155266300

I guess visual dividing logically separated portions of code is a good thing.
Anything is good if it adds readability.

Reply:
I think it's enough to have 1 line to have `visual dividing logically separated portions of code`

Reply:
What if portions to be separated have empty lines inside?

Reply:
I mean, these are mostly just functions and not various heterogenous chunks of code, so I don't really think that applies here. But 🤷, I don't really have a stake in this.